SportsHub/SportsHub.DAL/Repository/ArticleRepository.cs

```diff
@@ -17,5 +17,18 @@ public ArticleRepository(ApplicationDbContext context) : base(context)
                 .Where(x => x.Title == title)
                 .FirstOrDefaultAsync();
         }
+
+        public async Task<bool> DeleteArticleAsync(int id)
+        {
+            var article = await DbSet
```

Comment:
with that approach looks like the repository has a logic for when to delete, we need to avoid it. I meant: in your service action
1. calls repository action by Id
2. check the result if does not exist - then throw an exception - (when will be implemented, now it false)
3. if exist - calls repository Delete action
4. Save Async

Reply:
Okey i will change it tommorow after i got home from work before our meeting